### ISSUE TITLE
Worker SIGTERM after 1.12.11 upgrade

### ISSUE THREAD

**Issue by gabrielmustiere** [Sylius/Sylius#15401]
**Sylius version affected**: 1.12.11

After updating Sylius to 1.12.11, the container worker executing the `messenger:consume` command returns an error.

```
/srv/sylius # bin/console messenger:consume

In ConsumeMessagesCommand.php line 261:

  Undefined constant "SIGTERM"


messenger:consume [-l|--limit LIMIT] [-f|--failure-limit FAILURE-LIMIT] [-m|--memory-limit MEMORY-LIMIT] [-t|--time-limit TIME-LIMIT] [--sleep SLEEP] [-b|--bus BUS] [--queues QUEUES] [--no-reset] [--] [<receivers>...]
```

Rollback to 1.12.8 resolve the issue.

**Comment by gabrielmustiere** [comment]
add pcntl to `dockerfile` `RUN install-php-extensions apcu exif gd intl pdo_mysql opcache zip pcntl` resolve issue

**Comment by JoppeDC** [comment]
Going to comment here as well for completeness.

[This messenger PR](https://github.com/symfony/symfony/pull/50787) introduced the usage of `\SIGTERM` and `\SIGINT` constants. These are only available when the `pcntl` PHP extension is installed.

[In a later PR](https://github.com/symfony/symfony/pull/51791), a check was added to see if the extension is installed (and thus if the constants can actually be used).

Due to a patch version being recently released, `Symfony/Messenger:6.3.5` will always require the extension, as the fix was not yet available in this tag.

The next tagged version of Symfony Messenger should have this fix included.

**Comment by gabrielmustiere** [comment]
Thanks a lot
